connections.js

```diff
@@ -0,0 +1,64 @@
+import { MongoClient } from 'mongodb';
+import functions from './database.js';
+
+const uri = 'mongodb://localhost:27017';
+export const client = new MongoClient(uri);
+
+const { addNewSoldier, lookForSoldier, lookForAllSoldiers } = functions;
+
+async function connectToDB(soldier) {
+  try {
+    await client.connect().then(() => console.log('connected to db'));
+    await addNewSoldier(client, {
+      _id: soldier.id,
+      name: soldier.name,
+      rank: soldier.rank,
+      limitations: soldier.limitations,
+    });
+  } catch (e) {
+    console.log(e);
+    throw e;
+  } finally {
+    await client.close().then(() => console.log('closed'));
+  }
+}
+
+async function connectToDBToFind(findSoldier) {
+  try {
+    await client.connect();
+    return await lookForSoldier(
+      client,
+      { _id: parseInt(findSoldier, 10) },
+    );
+  } catch (e) {
+    console.log(e);
+    throw e;
+  } finally {
+    await client.close();
+  }
+}
+
+async function connectToDbToFinAll(findSoldiers) {
```

Comment:
connectToDbToFindAll*
Also, no need for another function, separate connect and findAll

Reply:
done